yarn-project/acir-simulator/src/avm/opcodes/memory.ts

```diff
@@ -7,12 +7,12 @@ export class Set extends Instruction {
   static type: string = 'SET';
   static numberOfOperands = 3;
 
-  constructor(private value: bigint, private dstOffset: number, private dstTag: TypeTag) {
+  constructor(private inTag: TypeTag, private value: bigint, private dstOffset: number) {
```

Comment:
Renamed flag to `inTag` to match YP. It is describing the type of the input constant, but it's also determining what tag to use for the destination.... :thinking: _Should_ it be `dstTag`? Not sure, but I guess either way let's keep it in sync with the YP.